clang/test/Sema/warn-stringcompare.c

```diff
@@ -15,15 +15,15 @@ void test(const char *d) {
   if (NULL == "/")
     return;
   if ("/" != DELIM) // expected-warning {{result of comparison against a string literal is unspecified (use an explicit string comparison function instead)}}
-    return;
+    return;         // cxx-warning@-1 {{comparison between two arrays}}
```

Comment:
This seems to overlap w/ `warn_stringcompare` we probably want to do some special handling here since string literals are a special case of arrays and they have their own comparison functions.

We can do this as part of the C++26 feature implementation.

Reply:
CC @erichkeane

Reply:
Oooh, thats a great concern!  I think you're right, but I think as well that it is ok to do in the followup